bundles/org.openhab.core.io.rest/src/main/java/org/eclipse/smarthome/io/rest/Stream2JSONInputStream.java

```diff
@@ -96,11 +95,7 @@ private void fillBuffer() {
         }
 
         IOUtils.closeQuietly(jsonElementStream);
-        try {
-            jsonElementStream = IOUtils.toInputStream(prefix + entity + postfix, StandardCharsets.UTF_8.name());
```

Comment:
Hm, I would have thought that there was a good reason to pin UTF-8 encoding here. After all, we are talking about the REST API, which should use a strictly defined encoding.
Are you sure that it isn't rather the test that might need to be modified?